x/merkledb/proof.go

```diff
@@ -845,7 +847,7 @@ func addPathInfo(
 	return nil
 }
 
-func getEmptyTrieView(ctx context.Context) (*trieView, error) {
+func getTrieView(ctx context.Context, ops []database.BatchOp) (*trieView, error) {
```

Comment:
Trying to think of a better name for this. `newStandaloneTrieView` or something? Idk.

Reply:
Maybe a comment would suffice.